INSTALL.asciidoc

```diff
@@ -25,16 +36,20 @@ sudo zypper in libtheora-devel opencv-devel fftw3-devel libsndfile-devel pkg-con
 autoreconf -f -i
 ./configure
 make
+make install
```

Comment:
I would mark that as optional. As having `os-autoinst` usable is basically having isotovideo available it is good enough to just build it and use the isotovideo as is after `make`.